src/main/java/underdogs/devbie/advice/ControllerAdvice.java

```diff
@@ -0,0 +1,45 @@
+package underdogs.devbie.advice;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import underdogs.devbie.advice.dto.ErrorResponse;
+import underdogs.devbie.exception.BadRequestException;
+import underdogs.devbie.exception.ForbiddenException;
+import underdogs.devbie.exception.IntervalServerException;
+import underdogs.devbie.exception.UnAuthorizedException;
+
+@RestControllerAdvice
+public class ControllerAdvice {
+
```

Comment:
```suggestion
@RestControllerAdvice
public class ControllerAdvice {

```
혹시 모를 에러에 대비해 Exception.class에 대한 ExceptionHandler를 만들어주는건 어떨까요?

Reply:
```java
	@ExceptionHandler(Exception.class)
    public ResponseEntity<ErrorResponse> unHandledExceptionHandler(Exception exception) {
        ErrorResponse errorResponse = new ErrorResponse(exception.getMessage());
        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
            .body(errorResponse);
    }
```

를 추가하였습니다.